tests/pl/weather_HassGetWeather.yaml

```diff
@@ -0,0 +1,25 @@
+language: pl
+tests:
+  - sentences:
+      - "jaka jest pogoda"
+    intent:
+      name: HassGetWeather
+    response: 8 °C i pada deszcz
+
+  - sentences:
+      - "jaka jest pogoda w Warszawa"
```

Comment:
The correct sentence should be "jaka jest pogoda w Warszawie"

Reply:
I'm not sure you can do automatic declination of entities. This could work with aliases though, but by using default entity names here you match the UI

Reply:
Yes, you're right but the future these declensions will be used to create sentences for the ML model. I would prefer it to sound correctly in Polish 😀 check this issue https://github.com/home-assistant/intents/issues/976